proto/api_v3/query_service.proto

```diff
@@ -40,47 +40,65 @@ message GetTraceRequest {
   bool raw_traces = 4;
 }
 
-// Query parameters to find traces. Except for num_traces, all fields should be treated
-// as forming a conjunction, e.g., "service_name='X' AND operation_name='Y' AND ...".
-// All fields are matched against individual spans, not at the trace level.
-// The returned results contain traces where at least one span matches the conditions.
-// When num_traces results in fewer traces returned, there is no required ordering.
+// Query parameters to find traces.
 //
-// Note: num_traces should restrict the number of traces returned, but not all backends
-// interpret it this way. For instance, in Cassandra this limits the number of _spans_
-// that match the conditions, and the resulting number of traces can be less.
+// All fields form a conjunction (e.g., "service_name='X' AND operation_name='Y' AND ..."),
+// except for `search_depth` and `raw_traces`.
 //
-// Note: some storage implementations do not guarantee the correct implementation of all parameters.
+// Fields are matched against individual spans, not the trace level. The results include
+// traces with at least one matching span.
 //
+// The results have no guaranteed ordering. Additionally, some storage implementations
+// may not fully support all parameters.
 message TraceQueryParameters {
+  // `service_name` filters traces generated by a specific service.
   string service_name = 1;
+
+  // `operation_name` filters traces generated by a specific operation.
```

Comment:
```suggestion
  // `operation_name` filters spans by a specific operation / span name.
```

Reply:
we could rename this to `span_name`, to match OTLP more closely. "operation" it not a term in OTEL.

Reply:
@yurishkuro For the rename, I assume you mean for remote storage api v2?